tests/playwright/propTests/raisedPropTest.ts

```diff
@@ -0,0 +1,8 @@
+import type { PropTests } from '../types';
+
+export const raisedPropTest: PropTests = [
+  {
```

Comment:
Please, follow the rules set in parent issue. Again, boolean values must have both values defined in prop tests.

Reply:
I am sorry if it's missing. Nevertheless, we do this for all the boolean values as propTests are independent to default component props.